packages/catalog-api/src/lib/schema.graphql

```diff
@@ -0,0 +1,250 @@
+type Query {
```

Comment:
I think we still have an exercise to talk through what the best API should be exactly, but I'm fine with landing this knowing that we can continue to iterate on it.